GliaWidgets/Sources/CallVisualizer/Coordinator/CallVisualizer.Coordinator.swift

```diff
@@ -34,6 +34,13 @@ extension CallVisualizer {
                 switch event {
                 case .closeTap:
                     self?.visitorCodeCoordinator = nil
+                case .engagementAccepted:
+                    switch coordinator.presentation {
+                    case let .embedded(_, onEngagementAccepted: callBack):
```

Comment:
`callback` is fine

Reply:
fixed [3461dd9](https://github.com/salemove/ios-sdk-widgets/pull/538/commits/3461dd9de31bdc991a93f12e25092d53f83d2fc6)